linux/dumper/dumper_lcd.py

```diff
@@ -26,9 +26,7 @@ def init_lcd():
     lcd_columns = 16
     lcd_rows = 2
     i2c = busio.I2C(board.SCL, board.SDA)
-    lcd = character_lcd.Character_LCD_RGB_I2C(i2c, lcd_columns, lcd_rows)
-
-    return lcd
+    return character_lcd.Character_LCD_RGB_I2C(i2c, lcd_columns, lcd_rows)
```

Comment:
This actually has to return `lcd` because it's what allows you to generate messages. Unless since we are decoupling that from `main`, we can instantiate the LCD in another file.